### PR TITLE
using currentColor as the default instead of #000

### PR DESCRIPTION
**Pull request by lenzi-erickson**
Changing the default fill color to currentColor instead of #000. Which will make the icons default to the text color unless a color prop is passed.

Issue: https://github.com/levrik/mdi-react/issues/32

### CODE REVIEW

**Comment by levrik** [comment]
Hey.
Thanks for this investigation. I tried this back then when I implemented the color attribute but for some reason `currentColor` as `fill` on the path didn't work. Just tried now, works perfectly fine.
Let me check if something's changed in between and if all major browsers show the same behavior.

**Comment by lenzi-erickson** [comment]
Awesome, thanks! This will be a huge help in our project. :)

**Comment by levrik** [comment]
In the meanwhile adding a
```css
.mdi-icon {
  fill: currentColor;
}
```
to your global CSS should work too.

**Comment by lenzi-erickson** [comment]
Any ETA on when this might get implemented? I'm using the workaround for now

```
.mdi-icon {
  fill: currentColor;
}
```

but then it overrides the color prop and you can no longer pass prop as a color. I was hoping to be able to have it default to the currentColor but still be able to pass a color prop to the icon. Any updates are appreciated. Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,6 @@ _No changes_
 ### New features
 
 - [`d5aac53`](https://github.com/levrik/mdi-react/commit/d5aac537dfcf800a1cdbc24975b4efcdb7766981) Added prop `size`. Default is `24px`.
-- [`544b865`](https://github.com/levrik/mdi-react/commit/544b865a886fd87233e8fe4e0201832bd39286db) Added prop `color`. Default is `#000`.
+- [`544b865`](https://github.com/levrik/mdi-react/commit/544b865a886fd87233e8fe4e0201832bd39286db) Added prop `color`. Default is `currentColor`.
```
Comment (levrik) [inline CHANGELOG.md]: Please don't edit the changelog

Comment (levrik) [inline CHANGELOG.md]: Please don't edit the changelog

Comment (lenzi-erickson) [reply CHANGELOG.md]: Done!

**Comment by shrshk** [comment]
waiting on this stuff to be merged to master.

**Comment by lenzi-erickson** [comment]
Any luck on this getting merged to master? I believe I made the change you requested. :)

**Comment by levrik** [comment]
@shrshk @lenzi-erickson Sorry. Was busy with work and health issues. Hopefully make a new release this weekend.

**Comment by levrik** [comment]
@lenzi-erickson @shrshk This is released in 4.0.0

**Pull request merged**